wire-schema/src/jvmMain/java/com/squareup/wire/schema/MarkSet.kt

```diff
@@ -52,6 +57,24 @@ class MarkSet(
     types.add(type)
   }
 
+  /**
+   * Marks a type as transitively reachable by the includes set. Returns true if the mark is new,
+   * the type will be retained, and reachable objects should be traversed.
+   * Even though `type` might be excluded, it will be retained if it is one the root member's
```

Comment:
Can I rephrase this sentence?

```
If there is an exclude for [type], non-root members referencing it will be pruned. The type itself will also be pruned unless is referenced by a root member.
```

Reply:
fixed